target/linux/ramips/mt7621/base-files/etc/board.d/02_network

```diff
@@ -32,6 +32,7 @@ ramips_setup_interfaces()
 	linksys,re6500)
 		ucidef_set_interface_lan "lan1 lan2 lan3 lan4"
 		;;
+	edimax,re23s|\
```

Comment:
This affects sorting, please move the block accordingly.

Reply:
Fixed, thanks